apps/E2E/src/Chip/pages/ChipPageObject.ts

```diff
@@ -13,9 +13,7 @@ class ChipPageObject extends BasePage {
   /* Waits for the text content to get updated to new string.
    * Returns true if new string is attained. */
   async waitForStringUpdate(newState: string, errorMessage: string): Promise<boolean> {
```

Comment:
Why do we need this wrapper function?

Why not just call `await this.waitForCondition(async () => await this.verifyTextContent(newState), errorMessage);` from the spec?

Reply:
I had added it initially to follow the same pattern as that followed in Input tests, but here since there is only one call made to this function, removing wrapper fn